manifests/0.5.1/3scale-community-operator.v0.5.1.clusterserviceversion.yaml

```diff
@@ -298,20 +317,23 @@ spec:
       mediatype: image/png
```

Comment:
Wait, shouldn't this be `image/svg+xml` ?
Also the base64data of the icon seems different than the productized CSV :thinking:

Reply:
Yes, it has been decided to be like that.

Reply:
I see :+1: we have different icons (and their types too) between community and productized